### ISSUE TITLE
Update ID vitals methodology and copy 

### ISSUE THREAD

**Issue by sychang** [Recidiviz/pulse-dashboard#1172]
**What needs to be done? Why does it need to be done?**
Update methodology and copy

TODO
- Methodology: https://docs.google.com/document/d/1moftAcmQ2JWddxwI99btG0DDSC8fBIChwfysb6Hvam4/edit#heading=h.zaj0dhwrazbu
- Copy updates: https://docs.google.com/document/d/1moftAcmQ2JWddxwI99btG0DDSC8fBIChwfysb6Hvam4/edit#heading=h.zaj0dhwrazbu

**Additional context**
Add any other context about the task here.

**Comment by jovergaag** [comment]
Metric description updates are done - included in https://github.com/Recidiviz/pulse-dashboard/pull/1176

**Comment by soniachokshi** [comment]
I just took a look at the copy in staging, and I noticed that there are actually a couple of small grammar errors. (Not on you, Jen; I think Serena accidentally wrote them this way in the doc.)

Here's what they should be:
- Timely assessment: XX% of clients have an up-to-date risk assessment, according to IDOC policy.
- Timely contacts: XX% of clients have an up-to-date contact, according to IDOC policy.
- Timely downgrades: XX% of clients are at a supervision level that is the same or below their risk level.
